packages/w3up-client/README.md

```diff
@@ -153,19 +136,61 @@ try {
 }
 ```
 
-By default, calling `registerSpace` registers the Space with web3.storage w3up. You can pass the optional `provider` param to register with NFT.Storage w3up instead.
+If you created the Space in w3cli using `w3 space create`, you can register the space by calling [`w3 space register`](https://github.com/web3-storage/w3cli#w3-space-register).
+
+#### Delegating from Space to Agent
+
+In order to store data with w3up, your Agent will need a delegation from a Space. The easiest way to do this is by authorizing your Agent. This allows you to claim delegations to the Space that is authorized to the _same_ email account. Authorization needs to happen only once per agent.
 
 ```js
-try {
-  await client.registerSpace('zaphod@beeblebrox.galaxy', { provider: 'did:web:nft.storage' })
-} catch (err) {
-  console.error('registration failed: ', err)
+await client.authorize('zaphod@beeblebrox.galaxy')
+```
+
+Calling `authorize` will cause an email to be sent to the given address. Once a user clicks the confirmation link in the email, the `authorize` method will resolve. Make sure to check for errors, as `authorize` will fail if the email is not confirmed within the expiration timeout.
+
+If this is not the first time you authorized an agent with your email, then you'll want to claim any spaces and delegations you have on your other agent(s):
+
+```js
+await client.capability.access.claim()
+```
+
+##### Bringing your own Agent and delegation
+
+For uses of `w3up-client` in environments where the Agent is not persisted and/or the email verification step would be prohibitive (e.g., serverless backend environment where local Store with the Agent is dropped in between runs, and going through the email verification flow isn't practical), you can manually add a delegation for access to a space created by a different authorized agent (see the [`addSpace` client method](docs-client#addSpace)). An example:
+
+```js
+import * as Signer from '@ucanto/principal/ed25519' // Agents on Node should use Ed25519 keys
+import { importDAG } from '@ucanto/core/delegation'
+import { CarReader } from '@ipld/car'
+import * as Client from '@web3-storage/w3up-client'
+
+async function main () {
+  // from "bring your own Agent" example in `Creating a client object" section`
+  const principal = Signer.parse(process.env.KEY) // created by `npx ucan-key ed --json` in command line, which returns private key and DID for Agent which were stored in environment variables KEY and PROOF
+  const client = await Client.create({ principal })
+  
+  // now give Agent the delegation from a Space created in w3cli using `w3 space create`
+  const proof = await parseProof(process.env.PROOF) // function created locally
```

Comment:
not sure what this means by "function created locally" - ideally this would point to docs for creating this proof (probably using w3cli)

Reply:
i think i included a note on the local function parseProof bc i couldn't find where it was imported from for like 5 minutes. but yes probably isn't needed for engineers!